pkg/cloudprovider/aws/suite_test.go

```diff
@@ -411,6 +418,20 @@ var _ = Describe("Allocation", func() {
 					aws.String("test-security-group-3"),
 				))
 			})
+			It("allow multiple security groups with cluster tag", func() {
```

Comment:
This unit test made more sense in an earlier code revision when I wanted to verify that multiple security groups with the cluster tag would lead to a runtime controller failure. I'm still leaving it in though because if we ever do decide to regress on this by re-adding that functionality, it should break.

Reply:
This is in the class of "inversion testing" or testing that things don't happen. I tend to lean away from it, since there are infinitely many things that we can't test in this class. 

It does look like we were missing a test for using the security group selector, so perhaps you can just rename the `It` block to something like `should discover security groups by tag`?

Reply:
Agreed, so just renamed the It block.